react/Modal/Readme.md

```diff
@@ -20,7 +20,7 @@ initialState = { modalDisplayed: false};
   <button onClick={()=>setState({ modalDisplayed: !state.modalDisplayed })}>
     Toggle modal
   </button>
-  {state.modalDisplayed && <Modal  description={content.ada.short} dismissAction={() => setState({ modalDisplayed: false })} /> }
+  {state.modalDisplayed && <Modal description={content.ada.short} arial-title="Accessible title" dismissAction={() => setState({ modalDisplayed: false })} /> }
```

Comment:
arial -> aria

Reply:
Wow that's a 100% pure *typo* 😆
Nice catch, thanks.